### PR TITLE
add & update links

### PR DESCRIPTION
**Pull request by davis278**
Added link to new notebook (VCS Basics); updated other links; provided .png files; made a few other text edits.

I tried to put @mauzey1 as a reviewer, but sadly Chris does not come up in my list of reviewers :(

### CODE REVIEW

File: tutorials.html [hunk 1]
```diff
@@ -49,20 +49,28 @@ <h3>CDMS</h3>
     <div class="container">
         <h3>VCS</h3>
         <p> Examples of how to use VCS</p>
+        <div class="example">
+            <a href="Jupyter-notebooks/vcs/VCS_Basics/VCS_Basics.html">
+                <div class="img-wrapper">
+                    <img src="VCS_Basics.png"/>
```
Comment (mauzey1) [inline tutorials.html]:
```suggestion
                    <img src="Jupyter-notebooks/vcs/VCS_Basics/VCS_Basics.png"/>
```

File: tutorials.html [hunk 2]
```diff
@@ -49,20 +49,28 @@ <h3>CDMS</h3>
...
+                </div>
+                <p>VCS Basics</p>
+            </a>
+        </div>
         <div class="example">
             <a href="Jupyter-notebooks/vcs/VCS_Principles/VCS_Principles.html">
                 <div class="img-wrapper">
-                    <img src="images/uvcdat.png"/>
+                    <img src="VCS_Principles.png"/>
```
Comment (mauzey1) [inline tutorials.html]:
```suggestion
                    <img src="Jupyter-notebooks/vcs/VCS_Principles/VCS_Principles.png"/>
```

File: tutorials.html (hunk 1 above)
Comment (davis278) [reply tutorials.html]: Thanks @Mauzey1. With the new .pngs on a different branch that had not been deployed when I first tested the page, it was more difficult for me to catch this mistake. I believe it should be fixed with the second commit for this PR.

File: tutorials.html (hunk 2 above)
Comment (davis278) [reply tutorials.html]: Thanks @Mauzey1. With the new .pngs on a different branch that had not been deployed when I first tested the page, it was more difficult for me to catch this mistake. I believe it should be fixed with the second commit for this PR.

**Pull request merged**